File: src/dotnet/Kurdi.ECommerce.Inventory.Core/Entities/ProductAggregate/ProductDetails.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;

namespace Kurdi.ECommerce.Inventory.Core.Entities.ProductAggregate
{
    [Table(name: "product_details")]
    public class ProductDetails
    {
        [Column(name: "language_code")]
        public string LanguageCode { get; set; }
        [ForeignKey(name: "LanguageCode")]
        public Language Language { get; set; }

        [Column(name: "sku")]
        public string Sku { get; set; }
        [ForeignKey(name: "Sku")]
        [JsonIgnore]
        public Product Product { get; set; }

        [Column(name: "name")]
        public string Name { get; set; }

        [Column(name: "description")]
        public string Description { get; set; }
        public TimeStamps TimeStamps { get; set; }

    }
}

File: core/entities/products/product_details.go
package products

import (
	"kurdi-go/core/entities"
)

type ProductDetails struct {
	Name         string `json:"name"`
	Description  string `json:"description"`
	LanguageCode string `json:"language_code"`
	StockItemId  int    `json:"stock_item_id"`
	entities.TimeStamps
}
